Configuration/StandardSequences/python/DigiToRaw_cff.py

```diff
@@ -19,8 +19,7 @@
 from L1Trigger.Configuration.L1TDigiToRaw_cff import *
 from EventFilter.CTPPSRawToDigi.ctppsDigiToRaw_cff import *
 
-#DigiToRawTask = cms.Task(L1TDigiToRawTask, siPixelRawData, SiStripDigiToRaw, ecalPacker, esDigiToRaw, hcalRawDataTask, cscpacker, dtpacker, rpcpacker, ctppsRawData, castorRawData, rawDataCollector)
-DigiToRawTask = cms.Task(L1TDigiToRawTask, siPixelRawData, SiStripDigiToRaw, ecalPacker, esDigiToRaw, hcalRawDataTask, cscpacker, dtpacker, rpcpacker, castorRawData, rawDataCollector)
+DigiToRawTask = cms.Task(L1TDigiToRawTask, siPixelRawData, SiStripDigiToRaw, ecalPacker, esDigiToRaw, hcalRawDataTask, cscpacker, dtpacker, rpcpacker, ctppsRawData, castorRawData, rawDataCollector)
```

Comment:
Does something actually consume the product(s) of `ctppsRawData` producer? (by quick `git grep ctppsRawData` I was not able to see any)

If nothing consumes `ctppsRawData`, it, and all modules whose products consumed only by it, are not run.

Reply:
Yes, but to do so, a flag, defined in g4SimHits_cff.py (LHCTransport = cms.bool(True)) needs to be set to true. This was implemented by @civanch to deal with the way PPS was downgrading the CMS simulation. In summary, it will be consumed for PPS simulation with the flago above set to true. Just to be 100% sure, I just tested it again, without the ctppsRawData there is no reco hits or tracks at the end.
Thanks.

Reply:
If I understand correctly, this is enable with `ctpps_2021` here
https://github.com/cms-sw/cmssw/blob/master/EventFilter/CTPPSRawToDigi/python/ctppsDigiToRaw_cff.py#L20

Reply:
Ah, right, `ctppsRawData` is a `Task`, and the relevant modules are consumed by the `rawDataCollector`. Thanks for the clarification, and sorry for the noise.

Reply:
No problem at all. Thank you.